.github/workflows/test.yml

```diff
@@ -23,7 +23,11 @@ jobs:
       # Remove node_modules to see if this action runs entirely compiled
     - name: 'Remove Node Modules'
       run: rm -rf node_modules
-    - name: 'Use this action'
+    - name: 'Test Working Directory Option'
```

Comment:
Should this test also have `if: always()` to ensure it runs even if tests fail (as the default action test has)?

Reply:
Good catch, but I don't think it's necessary here - as if the tests fail, the action won't probably work anyways. 

So even in the original job, this is kind of useless in most cases - I think it is a leftover from when I checked if the coverage would actually be reported correctly even if the tests were failing 😅